src/configuration.ts

```diff
@@ -54,9 +48,10 @@ export enum OverridesStatus {
 }
 
 export class Configuration {
-  private context: vscode.ExtensionContext;
```

Comment:
This was required so we have access to these fields from `src/extension.ts`. When we remove the temporary code, these fields can return to be private.